src/components/ui/Accordion/Accordion.test.tsx

```diff
@@ -0,0 +1,16 @@
+import { render, screen } from '@testing-library/react';
+import { Accordion } from './index';
+
+describe('Accordion', () => {
```

Comment:
I think we can remove tests for Accordion since this component is just a wrapper around MUI